client/static/js/board.js

```diff
@@ -1,26 +1,233 @@
 
-var Board = {
-    edges: null,
-    maxRows: 1000,
-    maxCols: 1000,
```

Comment:
Ahh, I see.  Is that what the `3000` values are below?  Maybe we could set those here as `canvasWidth`/`canvasHeight`, or something?  If not that's totally fine!